### PR TITLE
chore: Remove index hack, move canvas layout outside for readability

### PR DESCRIPTION
**Pull request by istarkov**
## Description

iframe now points to `/canvas` route
$ (splat) route has no UI, and now does only technical part.
_index route will be builder route on builder urls i.e. `{ProjectId}.apps.webstudio.is` (next PR)

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

File: apps/builder/app/routes/_canvas.canvas.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  return (
+    <ClientOnly fallback={<Body />}>
+      <Canvas params={params} imageLoader={imageLoader} />
+    </ClientOnly>
+  );
+};
+
+export default CanvasRoute;
+
+// Reduces Vercel function size from 29MB to 9MB for unknown reasons; effective when used in limited files.
```
Comment (kof) [inline apps/builder/app/routes/_canvas.canvas.tsx:61]: I wish we could keep this outside of the builder, somewhere that looks like a vercel folder or so in the root

Comment (kof) [reply apps/builder/app/routes/_canvas.canvas.tsx:61]: because really builder should know nothing about vercel

**Review by kof — APPROVED** [review]
approving, will do some more testing later

File: apps/builder/app/routes/_canvas.canvas.tsx (hunk 1 above)
Comment (istarkov) [reply apps/builder/app/routes/_canvas.canvas.tsx:61]: Its the only place for export, vercel related remix build

**Pull request merged**